lib/recipe/htmlToTextObjects.js

```diff
@@ -82,19 +82,22 @@ function parseNode(node) {
   const parsedData = {
     value,
     tag: node.tagName,
+    font: options ? options.font : undefined,
```

Comment:
Are these options in the typescript definitions?

Reply:
These options are not in the typescript definitions. They are similar to the objects that are returned from the function _makeTextObject in lib/receipt/text.js